executor/builder.go

```diff
@@ -1031,8 +1031,8 @@ func (b *executorBuilder) buildProjBelowAgg(aggFuncs []*aggregation.AggFuncDesc,
 	}
 
 	return &ProjectionExec{
-		baseExecutor: newBaseExecutor(b.ctx, expression.NewSchema(projSchemaCols...), projFromID, src),
-		//numWorkers:    b.ctx.GetSessionVars().ProjectionConcurrency,
```

Comment:
Why change happened?

Reply:
I commented out this line in https://github.com/pingcap/tidb/pull/8618 to pass leak test.

Reply:
got